src/containers/SearchPage/components/SearchResults.jsx

```diff
@@ -19,10 +25,22 @@ import { ArticleResultShape, LocationShape } from '../../../shapes';
 import { GraphqlResourceTypeWithsubtypesShape } from '../../../graphqlShapes';
 import SearchContextFilters from './SearchContextFilters';
 
+const contentTypeIcons = {
```

Comment:
Hadde det vært en ide å flytte dette til en `helper/constants` med parametere for ikonegenskaper? Noe av det samme må gjøres i et annet issue nemlig: https://waffle.io/NDLANO/Issues/cards/5a9fe18a4a98d1001c91a20d